extra-devel/avr-binutils/autobuild/beyond

```diff
@@ -0,0 +1,26 @@
+abinfo "Remove executables with default names"
+for elf in ar as nm objcopy objdump ranlib strip readelf; do
+	rm -v "${PKGDIR}${_PREFIX}/bin/${elf}"
+done
+
+abinfo "Remove bfd libdep for avr"
+rm -v "${PKGDIR}${_PREFIX}"/lib/bfd-plugins/libdep.so
+
+abinfo "Renaming documentation"
+for info in as bfd binutils gprof ld; do
+	mv -v \
+		"${PKGDIR}${_PREFIX}/share/info/${info}.info" \
+		"${PKGDIR}${_PREFIX}/share/info/avr-${info}.info"
+done
+
+abinfo "Removing locale"
+rm -rv "${PKGDIR}${_PREFIX}/share/locale"
+
+abinfo "Creating symlinks in /usr/bin"
+mkdir -vp "${PKGDIR}/usr/bin"
+pushd "${PKGDIR}${_PREFIX}/bin"
+for i in avr-*; do
+        ln -sv "${_PREFIX}/bin/${i}" "${PKGDIR}/usr/bin/$i"
```

Comment:
Use relative paths for symlinks.

Reply:
This will break symlinked `/bin/avr-ld`. For consistency I would suggest to retain absolute symlink here....